btcjson/chainsvrresults.go

```diff
@@ -207,6 +207,13 @@ type GetBlockChainInfoResult struct {
 	*UnifiedSoftForks
 }
 
+// GetBlockFilterResult models the data returned from the getblockfilter
+// command.
+type GetBlockFilterResult struct {
+	Filter string `json:"filter"`
+	Header string `json:"header"`
```

Comment:
**Nit:** Whenever possible, adding extra comments always helps, since it shows up beautifully on [pkg.go.dev](https://pkg.go.dev/github.com/onyb/btcd/btcjson?tab=doc).
```suggestion
	Filter string `json:"filter"`  // the hex-encoded filter data
	Header string `json:"header"`  // the hex-encoded filter header
```

Reply:
Good shout makes the result easier to consume. Didn't consider pkg.go.dev

Also added comments to `GetBlockFilterCmd` fields though they're not as visible through `rpcclient`.